lib/fedex/request/rate.rb

```diff
@@ -19,7 +20,7 @@ def process_request
             rate_details[:service_type] = rate_reply[:service_type]
             rate_details[:transit_time] = rate_reply[:transit_time]
             rate_details[:special_rating_applied] = rate_details[:special_rating_applied]
-            Fedex::Rate.new(rate_details)
+            Fedex::Rate.new(rate_details, request_xml: @request_xml, response_xml: @response_xml)
```

Comment:
Can you think of way we could not set the request/response on every rate? Potential memory issues could be introduced with this